docs/learn/prompts.md

```diff
@@ -172,11 +206,14 @@ Make sure when writing multi-line prompts with message roles that you start the
 )
 ```
 
+This ensures that the content for each role is properly formatted and aligned.
```

Comment:
nit: i think we should make it more clear that since we are dedenting on the user's behalf we need to make sure we don't include extra tabs and such